src/macros/externs.sem

```diff
@@ -0,0 +1,51 @@
+extern branchFactoryExtern: [createBranch:[address,address,bytes32]:int256, getLastBranch:[]:int256]
```

Comment:
This file was deleted, I'm guessing it got brought back during a merge/rebase?

Reply:
must've. Will re-delete